swap_meet/clothing.py

```diff
@@ -1,2 +1,29 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+
+class Clothing(Item):
+
+
+    def __init__(self, id=None, age=10, condition=0, fabric='Unknown'):
```

Comment:
[nit] `age=10` is a surprising default. Why are things aged 10 if no age is specified?